network-api/networkapi/wagtailpages/tests/buyersguide/test_products.py

```diff
@@ -233,6 +234,46 @@ def test_secondary_related_articles_no_related_articles(self):
 
         self.assertListEqual(result, [])
 
+    def test_product_with_enabled_category_shows_cta(self):
+        """
+        Testing that a product with any assigned category that has
+        show_cta=True displays the CTA as expected.
+        """
+        product_page = self.product_page
+
+        cat1 = BuyersGuideProductCategory.objects.create(name="Cat 1", show_cta=True)
+        category_orderable = ProductPageCategory(
+                product=product_page,
+                category=cat1,
+            )
+        category_orderable.save()
+        self.product_page.product_categories.add(category_orderable)
+        self.product_page.save_revision().publish()
+
+        response = self.client.get(product_page.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNotNone(response.context['featured_cta'])
+
+    def test_product_with_disabled_category_hides_cta(self):
+        """
+        Testing that a product where no assigned categories have show_cta=True
+        hides the CTA as expected.
+        """
+        product_page = self.product_page
+
+        cat2 = BuyersGuideProductCategory.objects.create(name="Cat 2", show_cta=False)
+        category_orderable = ProductPageCategory(
+                product=product_page,
+                category=cat2,
+            )
+        category_orderable.save()
+        self.product_page.product_categories.add(category_orderable)
+        self.product_page.save_revision().publish()
+
+        response = self.client.get(product_page.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNone(response.context['featured_cta'])
+
```

Comment:
**Question** Have you considered adding a test for a product with multiple categories where one has the `show_cta` flag set to `False` the other has it `True`?

Reply:
Added `test_product_page_cta_with_multiple_categories`